components/service/experiments/src/main/java/mozilla/components/service/experiments/Experiments.kt

```diff
@@ -87,12 +93,29 @@ open class ExperimentsInternalAPI internal constructor() {
         evaluator = ExperimentEvaluator(ValuesProvider())
         valuesProvider = ValuesProvider()
 
-        source = null
+        updater?.let {
+            it.source = null
+        }
         storage = null
 
         isInitialized = false
     }
 
+    internal fun getGlean(): GleanInternalAPI {
```

Comment:
nit: is it useful to annotate this with `VisibleForTesting` as well?

Reply:
Good point.